src/include/sof/spinlock.h

```diff
@@ -94,7 +94,7 @@ extern struct tr_ctx sl_tr;
 		} \
 	} while (0)
 
-#if CONFIG_DEBUG_LOCKS_VERBOSE
+#  if CONFIG_DEBUG_LOCKS_VERBOSE
```

Comment:
why the space here?

Reply:
It's just some indentation to help with the nesting

Try:
```
indent -ppi 2 -st src/include/sof/spinlock.h 
```

Reply:
So is this sort of indentation OK? I think it helps but I don't care that much, just let me know.

Reply:
Just isn't common in my experience in places like the kernel. @plbossart what are your thoughts

Reply:
@marc-hb this kind of indentation doesn't look very common. I would prefer not to do it. On the other hand the comments at the end of `#else` are really useful.